tests/src/test_swap_arr.cpp

```diff
@@ -0,0 +1,19 @@
+#include <pybind11/embed.h>
+#include <catch2/catch.hpp>
+
+#include "deeptime/markov/tram/tram.h"
+
+TEST_CASE("TRAM Swap", "[tram]") {
```

Comment:
while part of tram this really has nothing to do with tram, call the test case `ExchangeableArray` or similar?